content/2-2.md

```diff
@@ -1,44 +1,46 @@
 ---
-title: "[Draft] Module 2: Navigational Menus"
-permalink: /curricula/developing-accessible-content/navigational-menus/
-ref: /curricula/developing-accessible-content/navigational-menus/
+title: "[Draft] Module 2: Menus"
+permalink: /curricula/developer-modules/menus/
+ref: /curricula/developer-modules/menus/
 lang: en
 github:
   repository: w3c/wai-curricula
   path: content/2-2.md
 license: creative-commons
+acknowledgements: /curricula/acknowledgements/
 footer: >
   <p><strong>Date:</strong> $Date</p>
-  <p><strong>Editors: </strong>Daniel Montalvo and <a href="http://www.w3.org/People/shadi/">Shadi Abou-Zahra</a>. Contributors: <a href="https://www.w3.org/WAI/EO/EOWG-members">EOWG Participants</a>. </p>
+  <p><strong>Editors:</strong> Daniel Montalvo and <a href="http://www.w3.org/People/shadi/">Shadi Abou-Zahra</a>. Contributors: <a href="https://www.w3.org/WAI/EO/EOWG-members">EOWG Participants</a>. ACKNOWLEDGEMENTS lists contributors and credits.</p>
   <p>Developed by the Education and Outreach Working Group (<a href="http://www.w3.org/WAI/EO/">EOWG</a>). Developed with support from the <a href="https://www.w3.org/WAI/about/projects/wai-guide/">WAI-Guide Project</a> funded by the European Commission (EC) under the Horizon 2020 program (Grant Agreement 822245).</p>
 navigation:
-  previous: /curricula/developing-accessible-content/structure-and-semantics/
-  next: /curricula/developing-accessible-content/images-and-graphics/
+  previous: /curricula/developer-modules/page-structure/
+  next: /curricula/developer-modules/images/
 ---
  
 ## Introduction
 {:.no-display}
 
-Courses based on this module:
+Courses based on this module should:
```

Comment:
Is there potentially a need to include something about coding menus to be responsive to viewport size?

Reply:
I think we could add that thought to what we have now.

Current:

> * code menu styles so that menus appear and behave as required, for example on hover and focus

Proposed:

> * code menu styles so that menus appear and behave as required, for example in different viewport sizes, and on hover and focus

Reply:
That seems ok and I am happy if no one else flags it. I think it is quite a big issue though and it would be a shame to lose it within something else.

Reply:
Fair enough. One thing is that menus appear where expected and another thing is that they actually resize. Adding.

> * code menus so that they resize depending on different viewport sizes